### DISCUSSION TITLE
Printer Safety by Adjusting Parameters?

### DISCUSSION THREAD

**Discussion by Mortisshadow** [NuclearPhoenixx/Kobra_Neo#9] [Q&A] [answered]
Hello,

I am very interested in flashing the firmware, but I am afraid of the fact that it increases some parameters that are not found in the standard firmware, such as increased feedrate and X and Y acceleration. Is this still safe for the printer? Will it not shorten the lifespan when using different parameters than those advertised by Anycubic?

Best regards,  
Mortis

**Comment by NuclearPhoenixx** [comment, marked as answer]
Hi there,

the increased feed rate and acceleration actually **only** increase the **maximum** feed rate and acceleration that is possible via software.
The **actual** feed rate and acceleration are controlled via your slicer (e.g. Cura or PrusaSlicer).

It won't magically turn up everything, that would certainly mess up the prints. It's perfectly safe as long as **you** don't mess with settings you don't fully understand 😄 

Cheers!

**Reply by Mortisshadow** [reply]
Hi,

Thank you so much for your detailed explanation! I really appreciate you taking the time to clarify this for me. The custom firmware is truly impressive, and I thank you for making it!

Thanks again, and best regards!

Mortis
